src/assets/scripts/client/App.js

```diff
@@ -127,18 +127,20 @@ export default class App {
     loadAirlinesAndAircraft(airportLoadList, initialAirportIcao, initialAirportResponse) {
         const airlineListPromise = $.getJSON('assets/airlines/airlines.json');
         const aircraftListPromise = $.getJSON('assets/aircraft/aircraft.json');
+        const airportGuideListPromise = $.getJSON('assets/guides/guides.json');
```

Comment:
To avoid screwing around with `App` and the constructors, we could pass `this.contentQueue` in AppController to the Guide Controller's constructor, and then load it then, instead of at initial load.

What I'm thinking:

```javascript
// <AppController>
this.airportGuideInfoController = new AirportGuideInfoController(this.contentQueue);
// <AirportGuideInfoController>
loadGuideData(): void
const guidePromise = this.contentQueue.add({ url: `assets/guides/${nextIcao}`, immediate: true});
guidePromise.then((data, textStatus, jqXHR) => { // handle + fire a callback });
```

Reply:
something like that is a good idea here because the data we need to load is not needed before the app starts (like airports, airlines and aircraft).  Could you add an issue for that?  I'd like to be able to take our time on it and get this current thing completely working first.

Reply:
#1321